INSTALL.md

```diff
@@ -0,0 +1,71 @@
+# Installation
+
+The subscription-manager codebase is on [GitHub](https://github.com/candlepin/subscription-manager).
+The project is built for the latest versions of Fedora (and submitted to Fedora Updates), CentOS Stream and RHEL.
+
+To install subscription-manager, run
+
+```bash
+sudo dnf install subscription-manager
+```
+
+## Developer installation
+
+The process below has been tested on Fedora 36.
+Other Fedora versions or distributions may require some adaptation.
+
+1. First you need to install RPM packages to build and run subscription-manager binaries:
+
+   ```bash
+   sudo dnf install git gcc python3-devel openssl-devel glib2-devel \
+       python3-rpm python3-librepo libdnf-devel cmake
+   ```
+
+   <!-- libdnf-devel, cmake are required to build product-id plugin -->
+
+2. Install Fedora's subscription-manager RPM and packages required to run the test suite:
+
+   ***NOTE**: Installing `subscription-manager` package is strictly not necessary, but it will pull down all dependencies and create all the files used by subscription-manager.*
+
+   ```bash
+   sudo dnf install subscription-manager dbus-daemon langpacks-de langpacks-ja
```

Comment:
Maybe add here note that installing langpacks-de and langpacks-ja is no necessary, but it is only recommended in the case you want to test correctness of translation or displaying of translation.

Reply:
I don't think they are optional, as there are few tests that check things in `de_DE.utf8` and `ja_JP.UTF8` locales. Grep for `locale.setlocale` and `locale_context`.

Reply:
What tests do mean? Do you mean unit tests? I have never installed these langpacks RPMS and I have never had problems with unit tests.

Reply:
> Do you mean unit tests?

Yes, there are few unit tests that do checks with those locales.

Ah, actually we need the glibc langpacks, not the "general" langpacks -- so `glibc-langpack-de`, `glibc-langpack-en`, and `glibc-langpack-ja` -- see also our Containerfile:
https://github.com/candlepin/subscription-manager/blob/5fe842957e1257c6cc8fa7cedc3fd668e7824c9b/Containerfile#L18-L20